homeassistant/components/media_player/squeezebox.py

```diff
@@ -215,9 +215,17 @@ def async_update(self):
 
         try:
             self._status = response.copy()
-            self._status.update(response["remoteMeta"])
-        except KeyError:
-            pass
+        except AttributeError:
+            _LOGGER.error("Squeezebox object %s invalid status", self._name)
```

Comment:
It looks like a log error is already emitted when there's an error getting the status in LogitechMediaServer::async_query above. Instead of this first try/except AttributeError, can you add the following to catch an invalid status? The rest looks good.
```python
if response is False:
    return
```

See: https://github.com/home-assistant/home-assistant/pull/5566/files#diff-5f3aab5e3caa2ce6af27d1c83261e9c7R162 and others